mod_reforged/hooks/entity/tactical/actor.nut

```diff
@@ -152,7 +150,7 @@
 
 	q.isTurnDone = @(__original) function()
 	{
-		if (::Tactical.getNavigator().isTravelling(this)) return false;		// Copy&Paste of check from vanilla
+		if (::Tactical.getNavigator().isTravelling(this)) return false;		// Copy&Paste of check  vanilla
```

Comment:
weird choice of removing "from" here and leaving 2 spaces in the comment

Reply:
That was a mistake that somehow got into the commit. I've removed that change from this PR.